sambacc/container_dns.py

```diff
@@ -27,18 +29,22 @@
 
 
 class HostState:
-    def __init__(self, ref="", items=[]):
+    T = typing.TypeVar("T", bound="HostState")
+
+    def __init__(
+        self, ref: str = "", items: typing.Optional[list[HostInfo]] = None
+    ) -> None:
         self.ref = ref
-        self.items = items
+        self.items = items or []
```

Comment:
Consider having (align with other member-initialization in this PR):
```
       self.ref: str = ref
       self.items: list[HostInfo] = items or []
```

Reply:
updated.